apps/meteor/client/views/room/UserCard/UserCardWithData.tsx

```diff
@@ -64,30 +65,23 @@ const UserCardWithData = ({ username, target, rid, open, onClose }: UserCardWith
 		onClose?.();
 	});
 
-	const userActions = useUserInfoActions({ _id: user._id ?? '', username: user.username }, rid);
-	const { actions: actionsDefinition, menu: menuOptions } = useActionSpread(userActions);
+	const { actions: actionsDefinition, menuActions: menuOptions } = useUserInfoActions(
+		{ _id: user._id ?? '', username: user.username },
+		rid,
+	);
 
 	const menu = useMemo(() => {
 		if (!menuOptions) {
 			return null;
 		}
 
-		return (
-			<Menu
-				flexShrink={0}
-				maxHeight='initial'
-				mi='x2'
-				key='menu'
-				renderItem={({ label: { label, icon }, ...props }): ReactElement => <Option {...props} label={label} icon={icon} />}
-				options={menuOptions}
-			/>
-		);
-	}, [menuOptions]);
+		return <GenericMenu title={t('More')} key='menu' data-qa-id='menu' sections={menuOptions} placement='bottom-start' />;
+	}, [menuOptions, t]);
 
 	const actions = useMemo(() => {
-		const mapAction = ([key, { label, icon, action }]: any): ReactElement => (
-			<UserCard.Action key={key} label={label} aria-label={label} onClick={action} icon={icon} />
-		);
+		const mapAction = ([key, { content, icon, onClick }]: any): ReactElement => {
+			return <UserCard.Action title={content} key={key} label={content} aria-label={content} onClick={onClick} icon={icon} />;
```

Comment:
`UserCardAction` already takes care of `title` prop and makes it same as label, we should only require to pass `label`.